docs/sources/flow/reference/components/prometheus.remote_write.md

```diff
@@ -361,3 +361,72 @@ prometheus.remote_write "default" {
 `prometheus.remote_write` uses [snappy](https://en.wikipedia.org/wiki/Snappy_(compression)) for compression.
 
 Any labels that start with `__` will be removed before sending to the endpoint.
+
+## Data retention
+
+The `prometheus.remote_write` component reuses the Prometheus Write-Ahead Log
+(WAL) implementation for resiliency against network outages. The component
+buffers the received metrics to a WAL per configured endpoint so that the
+[queue shards][] can flush them to it.
```

Comment:
Where are the metrics being flushed? To the endpoint?

Reply:
Yes, but the sentence was reading a bit repetitive. Should I clarify here?